### DISCUSSION TITLE
is there a method to get only nsfw/racist/sexist etc jokes?

### DISCUSSION THREAD

**Discussion by koshyj8** [Sv443/JokeAPI#268] [Q&A] [answered]
im making a discord bot where if the members use the command in a way, there is a 100% chance they will get an nsfw  or racist joke. its the opposite of blacklist...it makes only those types of jokes appear

**Comment by Sv443** [comment, marked as answer]
Sorry for the very late reply. [I've thought about adding such functionality](https://github.com/Sv443/JokeAPI/issues/127#issuecomment-657267848) and might do it in v2.5
